src/nwbinspector/_formatting.py

```diff
@@ -44,8 +45,8 @@ class FormatterOptions:
     """Class structure for defining all free attributes for the design of a report format."""
 
     def __init__(
-        self, indent_size: int = 2, indent: Optional[str] = None, section_headers: list[str] = ["=", "-", "~"]
-    ):
+        self, indent_size: int = 2, indent: Optional[str] = None, section_headers: tuple[str, ...] = ("=", "-", "~")
+    ) -> None:
```

Comment:
This was an old whoopsie on my end

Cardinal sin of Python to set an argument to mutable default